### ISSUE TITLE
AdNauseam doesn't collect ads, but still blocks them.

### ISSUE THREAD

**Issue by cvazouras** [dhowe/AdNauseam#2111]
### Describe the issue

I noticed that AdNauseam doesn't always collect ads, even though it does block them. As seen in the screenshot in nytimes.com ADN shows that 0 ads are clicked and collected. But if I click on the UBlock icon on bottom left it will reveal that it does block 24 ads.
I think this also causes a problem with my ad vault because fewer ads are collected.

I created a new profile on Firefox, but the problem remains. I also checked on Chrome (having manually installed ADN) and it shows 0 ads collected too.

### Screenshot in which the issue can be seen

ADN showing 0 ads:
![Screenshot from 2022-06-13 03-43-22](https://user-images.githubusercontent.com/43879300/173261222-e27301ca-eed9-4956-bc1b-792ef5919f91.png)

Actual number of ads being blocked:
![Screenshot from 2022-06-13 03-43-39](https://user-images.githubusercontent.com/43879300/173261230-147d5887-14c4-47b7-ab1b-06a7624d17e8.png)

### Steps for anyone to reproduce the issue
I don't know how to explain really, sorry.

### Your settings

- OS/version: Linux Mint 20.3 Cinnamon
- Browser/version: Mozilla Firefox for Linux Mint mint-001 - 1.0     Version: 101.0.1
- AdNauseam version: v3.13.1
- Other extensions you have installed: Ctrl+NUmber to switch tabs, Facebook Container, HTTPS Everywhere, Netflix 1080p, Return YouTube Dislike, Translate Web Pages, To Google Translate, Search by Image, YouTube Channel Whitelist for uBlock Origin
- AdNauseam settings: I've only added *youtube.com/*&disableadblock=1 to my whitelist, so that I can allow ads on some youtube channels if I want to. Other than that, default settings.

##### Your filter lists
Default

### Your location/country
Greece

**Comment by mneunomne** [comment]
Thank you for your feedback @cvazouras. 

Unfortunatelly Adnauseam can't collect all the ads that it blocks. There are some limitatons such as video ads, some types of ads that are built dynamically by javaScript, and some iframe cases in which we can't have a specific enough query for collection. More infomration [here](https://github.com/dhowe/AdNauseam/wiki/Developer-FAQ#what-ads-does-adnauseam-collect)

But we are working on it! [About nytimes specifically](https://github.com/dhowe/AdNauseam/issues/1891)

**Comment by cvazouras** [comment]
Hm, I see. However is there any settings I can adjust so that more ads are collected? 
I think the number of ads collected is really low compared to the ones blocked. The screenshots above are from the new Firefox profile I created to test things out. But in my old profile, it says that since install, 103,087 ads were blocked, while the ad vault only has around 500 (can be seen in the previous screenshots, since I imported it).
![Screenshot from 2022-06-13 14-17-59](https://user-images.githubusercontent.com/43879300/173342909-e60279c5-9ecb-4af7-80a8-e4dc46e2295b.png)


Also, I've noticed this has been happening to other users for different reasons though. 
For example, #1437 and #1685

**Comment by mneunomne** [comment]
@cvazouras Yes indeed. This is part of the constant maintance that needs to be done in Adn. Ad networks are always changing together with their HTML structure, it's hard to keep up with all the change in the different ad providers. 

Along with that we try to make sure that Adn is as much of an efficient AdBlocker as uBlock Origin is, so we try to keep a balance but that requires constant work and re-work.

But thank you for your feedback, its an important reminder to what we need to be continuing doing.

**Comment by mneunomne** [comment]
Closing since the specific cases are being documented on other tickets.

**Comment by Meow-purr** [comment]
+1 i have it too
